### PR TITLE
[DOC] Add sphinx documentation

### PR DESCRIPTION
**Pull request by Herklos**
Available at https://docs.octobot.online

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+project = "OctoBot"
+copyright = "2021, DrakkarSoftware"
+author = "DrakkarSoftware"
+
+# The short X.Y version
+version = '0.4.0b2'
+
+# The full version, including alpha/beta/rc tags
+release = '0.4.0-beta2'
```
Comment (GuillaumeDSM) [inline docs/source/conf.py:19]: could we import / read octobot/constants.py for these ?

File: docs/source/pages/About.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.. role:: raw-html-m2r(raw)
```
Comment (GuillaumeDSM) [inline docs/source/pages/About.rst:1]: we will need to add a link to this page (I couldn't find any in the produced doc)

File: docs/source/pages/Home.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (GuillaumeDSM) [inline docs/source/pages/Home.rst:1]: what is the link to this page ?

File: docs/source/pages/Interfaces.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (GuillaumeDSM) [inline docs/source/pages/Interfaces.rst:1]: should we remove this page ?

File: docs/source/pages/Profiles.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.. role:: raw-html-m2r(raw)
```
Comment (GuillaumeDSM) [inline docs/source/pages/Profiles.rst:1]: Link to this page be added :)

File: docs/source/pages/TradingView-webhook.rst [hunk 6]
```diff
@@ -0,0 +1,95 @@
+.. role:: raw-html-m2r(raw)
+   :format: html
+
+
+TradingView webhook
+===================
+
+.. WARNING:: this page is only for OctoBot version >= 0.4.0
```
Comment (GuillaumeDSM) [inline docs/source/pages/TradingView-webhook.rst:8]: we should remove this warning

File: docs/source/pages/Webhooks.rst [hunk 7]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (GuillaumeDSM) [inline docs/source/pages/Webhooks.rst:1]: we should remove this page

File: docs/source/pages/Frequently-asked-questions-(FAQ).rst [hunk 8]
```diff
@@ -0,0 +1,70 @@
+.. role:: raw-html-m2r(raw)
+   :format: html
+
+
+Frequently asked questions
```
Comment (GuillaumeDSM) [inline docs/source/pages/Frequently-asked-questions-(FAQ).rst:5]: We should name it `Frequently Asked Questions (FAQ)`

File: docs/source/pages/With-Python-only.rst [hunk 9]
```diff
@@ -0,0 +1,39 @@
+With python only
```
Comment (GuillaumeDSM) [inline docs/source/pages/With-Python-only.rst:1]: With **P**ython only

**Review by GuillaumeDSM — COMMENTED** [review]
amazing job ! it will be so much easier to maintain and update the doc !

File: docs/source/pages/About.rst (hunk 2 above)
Comment (Herklos) [reply docs/source/pages/About.rst:1]: Yes, I didn't want to put it in sidebar, do you see a place for it ?

File: docs/source/pages/Home.rst (hunk 3 above)
Comment (Herklos) [reply docs/source/pages/Home.rst:1]: I think there is no

File: docs/source/pages/Interfaces.rst (hunk 4 above)
Comment (Herklos) [reply docs/source/pages/Interfaces.rst:1]: Yes :+1:

File: docs/source/pages/TradingView-webhook.rst (hunk 6 above)
Comment (Herklos) [reply docs/source/pages/TradingView-webhook.rst:8]: Yes

File: docs/source/pages/Profiles.rst (hunk 5 above)
Comment (Herklos) [reply docs/source/pages/Profiles.rst:1]: Yes :+1:

File: docs/source/pages/Webhooks.rst (hunk 7 above)
Comment (Herklos) [reply docs/source/pages/Webhooks.rst:1]: Yes

File: docs/source/pages/With-Python-only.rst (hunk 9 above)
Comment (Herklos) [reply docs/source/pages/With-Python-only.rst:1]: Yes

File: docs/source/pages/Frequently-asked-questions-(FAQ).rst (hunk 8 above)
Comment (Herklos) [reply docs/source/pages/Frequently-asked-questions-(FAQ).rst:5]: Yes

File: docs/source/conf.py [hunk 10]
```diff
@@ -1,22 +1,23 @@
 import os
-
 import sys
 
+import octobot.constants as constants
```
Comment (GuillaumeDSM) [inline docs/source/conf.py:4]: 👍

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**